src/views/product-landing/components/hero-heading-visual/index.tsx

```diff
@@ -11,12 +11,13 @@ function HeroHeadingVisual({
 		start: `var(--token-color-palette-neutral-100)`,
 		stop: `var(--token-color-palette-neutral-50)`,
 	}
-	const gradient = productSlug
-		? {
-				start: `var(--token-color-${productSlug}-gradient-faint-start)`,
-				stop: `var(--token-color-${productSlug}-gradient-faint-stop)`,
-		  }
-		: gradientDefault
+	const gradient =
+		productSlug !== 'hcp'
```

Comment:
⛏️  I think we still want to use `gradientDefault` if `productSlug` is falsy:

```suggestion
		productSlug && productSlug !== 'hcp'
```